### ISSUE TITLE
audio-features endpoint deprecated 

### ISSUE THREAD

**Issue by kampelmuehler** [JohanVanHoye/spotify-running-playlist-maker#2]
Unfortunately, just days before I found this tool, Spotify decided to deprecate the audio-features API endpoint and hence we can no longer filter songs for their features. What a pity.

**Comment by JohanVanHoye** [comment]
Yes, for quite some time this project has been fully unusable. I've finally managed to work around it by chaining together various external sources (last.fm, deezer next to spotify, and local BPM analysis).

**Comment by kampelmuehler** [comment]
Nice work!
I also then came up with my own approach built on top of an older spotify dump found here: https://www.kaggle.com/datasets/rodolfofigueroa/spotify-12m-songs

I've since switched to Tidal though.
